tests/testthat/test-layers-tfp.R

```diff
@@ -0,0 +1,139 @@
+
+# These tests correspond to (possibly shortened and/or modified versions of) the "docstring" tests in
+#
+# https://github.com/tensorflow/probability/blob/master/tensorflow_probability/python/layers/distribution_layer_test.py
+#
+# We don't test for correctness of the implementations as we're just consumers.
+# Instead, we test that we can successfully use these layers in Keras models from R.
+
+
+context("tensorflow probability layer methods")
+
+source("utils.R")
+
+library(tensorflow)
```

Comment:
We unfortunately can't execute this code at the top level of a test file (since these tests need to run on CRAN, which has no TF installed). Perhaps use an active binding?